config/vufind/config.ini

```diff
@@ -28,7 +28,12 @@ url             = http://library.myuniversity.edu/vufind
 ; make sure your reverse proxy sets the necessary headers.
 ;reverse_proxy    = true
 email           = support@myuniversity.edu
+; The title of your site, used in some messages and (by default) page <title> tags.
+; If you wish to remove this from titles, you can customize the title_wrapper
+; translation in your language files to remove the %%siteTitle%% token.
```

Comment:
It might be more clear why this changed by adding an explanation what the site title is used for in the title tags.

e.g.:
```
; This was added in order to improve accessibility for browser tab navigation with screen readers.
```